changelogs/client_server/newsfragments/1658.clarification

```diff
@@ -0,0 +1 @@
+event_annotations: annotion => annotation
```

Comment:
```suggestion
Fix typo in "event annotations" section: annotion => annotation.
```

Reply:
Actually for this sort of thing we normally just use

```suggestion
Fix various typos throughout the specification.
```

(See https://spec.matrix.org/v1.8/changelog/#client-server-api for example)